### PR TITLE
OCPBUGS-4615: Fix rendering variables in instructions

### PR DESCRIPTION
**Pull request by Vincent056**
This is in addition to https://github.com/ComplianceAsCode/compliance-operator/pull/191.

This adds variable rendering in instructions.

### CODE REVIEW

File: pkg/utils/parse_arf_result.go [hunk 1]
```diff
@@ -585,11 +585,11 @@ func complianceCheckResultRationale(rule *xmlquery.Node, valuesList map[string]s
 func getElementText(nptr *xmlquery.Node, elem string, valuesList map[string]string) (string, error) {
 	elemSelected := nptr.SelectElement(elem)
 	if elemSelected != nil {
-		rationaleRendered, _, err := RenderValues(XmlNodeAsMarkdownPreRender(elemSelected, true), valuesList)
+		eleRendered, _, err := RenderValues(XmlNodeAsMarkdownPreRender(elemSelected, true), valuesList)
```
Comment (jhrozek) [inline pkg/utils/parse_arf_result.go:588]: What does `eleRendered` mean?

**Review by jhrozek — COMMENTED** [review]
The code looks good with one question. I haven't checked the CI failures, just read the code.

**Comment by Vincent056** [comment]
retest

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**